backend/app/resources.py

```diff
@@ -0,0 +1,109 @@
+from flask_restful import Resource, marshal_with, marshal
+from sqlalchemy import exc
+
+from app.marshallers import item_marshaller, category_marshaller, subcategory_marshaller
+from app.models import Item, Category, Subcategory
+from app.parsers import item_parser, subcategory_parser, category_parser, creating_item_parser, \
+    creating_category_parser, creating_subcategory_parser
+from init_app import db
+
+
+def add_to_database(item):
```

Comment:
as I said before, this is the repository function example.

Reply:
It's C# example but you can do something similar 
https://docs.microsoft.com/en-gb/dotnet/standard/microservices-architecture/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design?view=aspnetcore-2.1#the-repository-pattern